demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -50,7 +50,7 @@ import {
 } from "./ProductForm.gql.generated";
 
 const rootBlocks = {
-    image: DamImageBlock,
+    image: PixelImageBlock,
```

Comment:
this change is wrong, but that is beacuse it's defined wrong in ProductForm.cometGen.ts

Reply:
I fixed it in this PR (required to make it build)